wgsl/index.bs

```diff
@@ -922,8 +922,11 @@ An attribute [=shader-creation error|must not=] be specified more than once per
   <tr><td><dfn noexport dfn-for="attribute">`workgroup_size`</dfn>
     <td>One, two or three parameters.
 
-    Each parameter [=shader-creation error|must=] be a a[=const-expression=] or an [=override-expression=].
+    Each parameter [=shader-creation error|must=] be a [=const-expression=] or an [=override-expression=].
     All parameters [=shader-creation error|must=] be the same type, either [=i32=] or [=u32=].
+
+    It is a [=pipeline-creation error=] unless each specified parameter is at least 1 and at most an upper
```

Comment:
This wording is confusing: interaction between unless (which is a negation), and how far that negation extends.

Suggest this construction instead, which is all positively-phrased conditions:

> A pipeline creation error results if any specified parameter is 0 or exceeds an upper bound specified by the WebGPU API.

We can be more specific:

> A pipeline creation error results if any specified parameter is 0 or exceeds the upper bound specified by the WebGPU API.

There are still further constraints about total number of invocations in the workgroup too.  Seems we should mention them or at least indicate they exist.

See https://gpuweb.github.io/gpuweb/#dom-gpudevice-createcomputepipeline 

Maybe add:

> Additionally, a pipeline creation error results if the product of the parameters exceeds the  [maxComputeInvocationsPerWorkgroup](https://gpuweb.github.io/gpuweb/#dom-supported-limits-maxcomputeinvocationsperworkgroup) device limit in WebGPU.

I'll request this change for a future CL

Reply:
heh.  Merged by accident even though I wanted the first wording change.

Will follow up immediately.